bin/drop_sample_annot.py

```diff
@@ -50,9 +54,11 @@ def __init__(self, bam, sample, strandedness, single_end, gtf, count_file, out_f
                 writer.writerow(sa_dict)
 
 
-def final_annot(count_file: Path, ref_annot: Path, out_file: Path):
-    """Concatenates the Sample Annotation produced by SampleAnnotation with the one
-    provided for the reference samples, checking for duplicate sample IDs"""
+def final_annot(count_file: str, ref_annot: str, out_file: str):
+    """
+    Concatenates the Sample Annotation produced by SampleAnnotation with the one
+    provided for the reference samples, checking for duplicate sample IDs
+    """
     df_samples = pd.read_csv("drop_annotation_given_samples.tsv", sep="\t")
     df_reference = pd.read_csv(ref_annot, sep="\t")
     df_reference["GENE_COUNTS_FILE"] = count_file
```

Comment:
is it
`df_reference = df_reference[df_reference["RNA_ID"].str.contains(id) == False]`
or
`df_reference = df_reference[df_reference["RNA_ID"].str.contains(id) == "False"]`

Reply:
It would be the first one, it will check if id is already part of RNA_ID and remove it from the pd.DataFrame

Reply:
👍